src/lib/cloud-manager-hoc.jsx

```diff
@@ -57,7 +57,7 @@ const cloudManagerHOC = function (WrappedComponent) {
         shouldConnect (props) {
             return !this.isConnected() && this.canUseCloud(props) &&
                 props.isShowingWithId && props.vm.runtime.hasCloudData() &&
-                props.canModifyCloudData;
+                props.canModifyCloudData && !props.vm.extensionManager.isExtensionLoaded('videoSensing');
```

Comment:
My only question with this is, which logic belongs in `shouldConnect()` and which in `mapStateToProps()`, where `canModifyCloudData` is defined? Maybe that calls for a larger code change, but it seems like we've left the logic scattered, instead of either a) in one place, or b) separated into distinct groups of related concerns.

Reply:
One more thought -- let's test this and make sure it seems to stop updating cloud vars if the video sensing extension becomes loaded. Maybe we need a change in the shouldDisconnect() function too?

Reply:
I didn't think about putting it in `mapStateToProps()`; that seems like a good idea. I think I will merge as-is so that this can move along, but I'll try to get that in a later update.